namerd/iface/interpreter-thrift/src/test/scala/io/buoyant/namerd/iface/ThriftNamerInterfaceTest.scala

```diff
@@ -16,7 +16,7 @@ import org.scalatest.FunSuite
 class ThriftNamerInterfaceTest extends FunSuite with Awaits {
   import ThriftNamerInterface._
 
-  def retryIn() = 1.second
+  val retryIn = () => 1.second
```

Comment:
`Eta-expansion of zero-argument method values is deprecated.`

It works if we replace usages of `retryIn` with `retryIn _` but I figured just val'ing up the function to begin with would be easier to understand.